src/lib.rs

```diff
@@ -49,7 +49,4 @@ fn total_fragments_needed(payload_length: u16, fragment_size: u16) -> u16
 {
     let remainder = if payload_length % fragment_size > 0 { 1 } else { 0 };
      ((payload_length / fragment_size) + remainder)
-}
-
-pub use net::{NetworkConfig, UdpSocket};
-pub use packet::Packet;
```

Comment:
Why are you removing those again? They're so common I thought they make sense here.

Reply:
I did that to make the user API better. Like the standard lib has:

use laminar::net::{all socket stuff}
use laminar::packet::{packet stuff}

I don't like it when we re export some types and other types not. Also when importing those types I want to be consistent instead of using super:: Packet for example.